packages.config

```diff
@@ -27,13 +28,13 @@
   <package id="BenchmarkDotNet" version="0.9.8"/>
   <package id="BenchmarkDotNet.Diagnostics.Windows" version="0.9.8"/>
   <package id="VisualCppTools" version="14.0.24519-Pre"/>
-  <package id="Newtonsoft.Json" version="6.0.8"/>
+  <package id="Newtonsoft.Json" version="8.0.1"/>
```

Comment:
i was consistently getting warning messages during the build that this specific version couldn't be found